src/utils/styleVariables.js

```diff
@@ -58,13 +65,25 @@ export default {
   size_large: pxToEm(40),
   size_jumbo: pxToEm(52),
 
-  spacing_quarter: pxToEm(2),
-  spacing_half: pxToEm(4),
-  spacing_single: pxToEm(8),
-  spacing_oneAndAHalf: pxToEm(12),
-  spacing_double: pxToEm(16),
-  spacing_triple: pxToEm(24),
-  spacing_quad: pxToEm(32),
+  space_inline_xxs: spacing_quarter,
+  space_inline_xs: spacing_half,
+  space_inline_sm: spacing_single,
+  space_inline_md: spacing_double,
+  space_inline_lg: spacing_triple,
+  space_inline_xl: spacing_quad,
+  space_inline_xxl: pxToEm(64),
```

Comment:
why not variable-ize this one too?

Reply:
I suppose we could, but it'd be something like `spacing_octo`?

Reply:
spacing8x?

Reply:
Good call. I went with:

```
const spacing_quarter = pxToEm(2);
const spacing_half = pxToEm(4);
const spacing_1x = pxToEm(8);
const spacing_2x = pxToEm(16);
const spacing_3x = pxToEm(24);
const spacing_4x = pxToEm(32);
const spacing_8x = pxToEm(64);
```